routing/unified_policies.go

```diff
@@ -208,6 +208,7 @@ func (u *unifiedPolicy) getPolicyNetwork(
 		bestPolicy  *unifiedPolicyEdge
 		maxFee      lnwire.MilliSatoshi
 		maxTimelock uint16
+		maxCapMsat  lnwire.MilliSatoshi
```

Comment:
👍 here for seeking out the max size given multiple chans and non strict forwarding and such